plugins/GSdx/GSCaptureDlg.cpp

```diff
@@ -113,83 +136,89 @@ void GSCaptureDlg::OnInit()
 		ComboBoxAppend(IDC_CODECS, s.c_str(), (LPARAM)&m_codecs.back(), c.DisplayName == selected);
 	}
 	EndEnumSysDev
+	UpdateConfigureButton();
 }
 
 bool GSCaptureDlg::OnCommand(HWND hWnd, UINT id, UINT code)
 {
-	if(id == IDC_BROWSE && code == BN_CLICKED)
+	switch (id)
```

Comment:
I'm not sure this change to switch + misc formatting is really worth it. If you have to do it then I'd prefer you put the change in a separate commit.